tests/loader_layer_tests.cpp

```diff
@@ -1635,11 +1635,27 @@ TEST_F(LayerExtensions, ImplicitNoAdditionalDeviceExtension) {
     dev.create_info.add_device_queue(DeviceQueueCreateInfo{}.add_priority(0.0f));
     dev.CheckCreate(phys_dev);
 
+    // Instance functions should be non-NULL since this could be available for any attached physical device.
```

Comment:
I read this as "Instance functions" implying the following functions are instance functions. Should probably read something like "Device function queried with GetInstanceProcAddr ..."